calcom/cal.com
# Variables for custom templates in workflows

## What does this PR do?

New feature to make custom workflow templates more flexible by adding variables. 
Fixes #3378

 Loom Video: https://www.loom.com/share/3351b21bd9f546fe9432c4c49227c8d5

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?

- Set up Sendgrid for email reminders (https://github.com/calcom/cal.com#setting-up-sendgrid-for-email-reminders)
- Create Workflow: 
   - Trigger: When event is booked
   - Action: Send Email to Attendee
   - Choose an event type in 'active on' 
   - Choose custom template and add a text with variables (see loom video)
   - Book an event with the event type that is active 
   - Check email that is sent to the attendee
   
## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
